src/construction.cpp

```diff
@@ -1520,8 +1520,12 @@ void finalize_constructions()
         }
     }
 
+    // previously dead code became live in #17451 and caused #17443 and #17449
+    // is temporarily disabled pending refactor as construction_id = string_id<construction>
+    /*
```

Comment:
Commented out code tends to stay there forever (look at mapgen.cpp, search for "Generate bodies / zombies", other examples exist as well). If the code is correct, keep it active, if it's incorrect, remove it. Otherwise it's just meaningless noise.

If something needs to be done at that place (removing now invalid constructions), add a TODO comment or similar there.